robot.html

```diff
@@ -809,6 +809,32 @@ <h1><a href="index.html" style="text-decoration:none;">Robotic Manipulation</a><
   appendix</a>.</p>
   
   </section>
+
+  <section><h1>Exercises</h1>
+
+    <exercise><h1>Role of Reflected Inertia</h1>
+
+      <p> For this exercise you will investigate the effect of reflected inertia on the joint-space dynamics of the robot, and how it affects simple position control laws. You will work exclusively in <a href="https://colab.research.google.com/github/RussTedrake/manipulation/blob/master/exercises/robot/reflected_inertia.ipynb" target="_blank">this notebook</a>. You will be asked to complete the following steps. </p>
+
+      <ol type="a">
+        <li> Derive the first-order state-space dynamics $\dot{\bx} = f(\bx, \bu)$ of a simple pendulum with a motor and gearbox. 
+        </li>
+        <li> Compare the behavior of the direct-driven simple pendulum and the simple pendulum with a high-ratio gearbox, under the same position control law.</li>
+      </ol>
+    </exercise>
+
+    <exercise><h1>The Manipulation Station</h1>
```

Comment:
How about "Input and Output Ports on the Manipulation Station"

Reply:
@RussTedrake changed in the new file! (sorry for the rebase)